### DISCUSSION TITLE
hybrid line numbers  not working

### DISCUSSION THREAD

**Discussion by fvasquezl** [LazyVim/LazyVim#3947] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/1359476/d9b65034-6e86-42d8-a5d5-3a2ad3c5846c)
As you see the line numbers not display the relative numbers, my LazyHealth only have something warnings in the key-maps, I'm using archlinux. I hope some suggestion to fix this issue

**Comment by dpetka2001** [comment]
What's your Neovim version (`nvim --version`)?

**Reply by fvasquezl** [reply]
![image](https://github.com/LazyVim/LazyVim/assets/1359476/743f179f-f2b9-4808-bbb4-50d12e8a7b92)

This is the version that is running

**Reply by dpetka2001** [reply]
It's quite old and there was a change in nightly that changed the way numbers are shown in the statuscolumn. So try to update Neovim to latest nightly and see if that solves your problem.

**Reply by fvasquezl** [reply]
Thank you very much, with that it is already working
